packages/amplify-graphql-conversation-transformer/src/transformer-steps/conversation-prepare-handler.ts

```diff
@@ -65,15 +66,17 @@ export class ConversationPrepareHandler {
    */
   private prepareResourcesForDirective(directive: ConversationDirectiveConfiguration, ctx: TransformerPrepareStepContextProvider): void {
     // TODO: Add @aws_cognito_user_pools directive to send messages mutation
```

Comment:
Is this TODO covered by [this change](https://github.com/aws-amplify/amplify-category-api/pull/2986/files#diff-cd6ca8120bf9ee84bff7871858a03c0f47fe68b128714b23f7ef779dda2d3959R152)?